plejd/PlejdAddon.js

```diff
@@ -126,6 +126,14 @@ class PlejdAddon extends EventEmitter {
       },
     );
 
+    this.plejdDeviceCommunication.on(PlejdDeviceCommunication.EVENTS.buttonPressed, (data) => {
```

Comment:
these can't unfortunately be strictly typed (or at least not enforced), so for me it is preferable to add named variables if possible (rather have a few params named for example `deviceId, ...`)

Reply:
In honestly do not know how to do that. Can you give an example. Adding `{deviceId, deviceInput}` was not correct.

Reply:
Assuming that is what you want to send, I would
* Set params as `(deviceId, inputIndex)`
* (match the buttonPressed call later)
* Update in `PlejdDeviceCommunication` row 120 and instead of `, data` send (guessing here) `, sourceDevice.deviceId, sourceDevice.input`
* Potentially remove the `data` variable in file above if not needed

Reply:
Fixed in coming updated